### PR TITLE
contrib: Add script to bump stable docker image tags

### PR DESCRIPTION
**Pull request by joestringer**

### CODE REVIEW

**Review by tklauser — APPROVED** [review]

File: contrib/release/bump-docker-stable.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    if ! echo "$1" | grep -q "[0-9]\+\.[0-9]\+\.[0-9]\+"; then
+        usage 2>&1
+        common::exit 1 "Invalid VERSION ARG \"$1\"; Expected X.Y.Z"
+    fi
+}
+
+main() {
+    handle_args "$@"
+
+    local ersion="$(echo $1 | sed 's/^v//')"
```
Comment (rolinh) [inline contrib/release/bump-docker-stable.sh:38]: :joy:

**Review by rolinh — APPROVED** [review]

**Review by aanm — CHANGES_REQUESTED** [review]
@joestringer I usually change the build setup on docker hub to build the stable tag the same time we are building a "normal" tag. Isn't that enough?

**Comment by joestringer** [comment]
@aanm I didn't want to click all the buttons to make that happen :joy: Did we ever figure out a way to automate all the dockerhub UI interactions?

**Comment by aanm** [comment]
> @aanm I didn't want to click all the buttons to make that happen  Did we ever figure out a way to automate all the dockerhub UI interactions?

Yes, once GH actions are done for `cilium/cilium` we should be good for >=1.9

**Comment by joestringer** [comment]
@aanm is that tracked somewhere specifically and is it planning to address both the v1.X.Y release image building / triggering as well as updating the `stable` tags? 

If the `stable` tag update will be handled by some other logic for v1.9 then I'll drop this PR.

**Comment by aanm** [comment]
> @aanm is that tracked somewhere specifically and is it planning to address both the v1.X.Y release image building / triggering as well as updating the `stable` tags?
> 
> If the `stable` tag update will be handled by some other logic for v1.9 then I'll drop this PR.

We'll have to double check this once we have it running

**Comment by joestringer** [comment]
@aanm oh, I guess there was a disconnect here. I marked this as draft under the assumption it's not necessary at this point. But it also has `priority/release-blocker`. Maybe I'll revert from draft again and we can evaluate whether to accept or just close it out.

**Comment by aanm** [comment]
@joestringer I might have marked this PR as release-blocker by accident

**Comment by joestringer** [comment]
@aanm given that we didn't address this for 1.9, would it be worthwhile for me to re-spin this PR to merge? There's now eight different images to configure the build targets on dockerhub each time.

**Comment by aanm** [comment]
> @aanm given that we didn't address this for 1.9, would it be worthwhile for me to re-spin this PR to merge? There's now eight different images to configure the build targets on dockerhub each time we do a 1.9.x release.

@joestringer it depends, as soon we have the GH actions builds done, we don't need to configure anything on docker hub anymore.

**Comment by joestringer** [comment]
@aanm that answer has not changed for the last six weeks. Do we have concrete expectations that those steps will occur in the short term? If not, I suggest we merge this as-is and revert whenever that system is fully set up.

**Comment by aanm** [comment]
> @aanm that answer has not changed for the last six weeks. Do we have concrete expectations that those steps will occur in the short term? If not, I suggest we merge this as-is and revert whenever that system is fully set up.

@joestringer I thought you had to make further changes to this PR if that's not the case I'm fine to merge it as is

**Comment by joestringer** [comment]
:+1: Just needed a rebase, already done.

**Pull request merged**